src/os/windows/mod.rs

```diff
@@ -178,8 +178,8 @@ impl Library {
     ///
     /// # Safety
     ///
-    /// When a library is loaded initialization routines contained within the library are executed.
-    /// For the purposes of safety, execution of these routines is conceptually the same calling an
+    /// When a library is loaded initialisation routines contained within the library are executed.
```

Comment:
Doesn't this need a "the" or a comma somewhere in between "loaded" and "initialization"?